packages/manager/modules/cloud-universe-components/src/rbx-eol-banner/translations/Messages_fr_FR.json

```diff
@@ -0,0 +1,3 @@
+{
+  "rbx1_eol_banner": "ATTENTION: Pour permettre la modernisation du bâtiment, les serveurs situés au Datacenter Roubaix RBX1 doivent être migrés avant le 31 décembre 2024. Merci de vous référer au mail reçu si vous avez des serveurs concernés. <a href=\"{{link}}\" target=\"_blank\">Consultez notre guide de migration.</a>"
```

Comment:
**nitpick:** Link can be separated into a different key and define the anchor tag in template.